apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/constant/PermissionType.java

```diff
@@ -1,27 +1,32 @@
 package com.ctrip.framework.apollo.portal.constant;
 
+/**
+ * 权限类型
+ */
 public interface PermissionType {
 
+
   /**
-   * APP level permission
+   * 应用级别的权限
    */
-
   String CREATE_NAMESPACE = "CreateNamespace";
-
   String CREATE_CLUSTER = "CreateCluster";
 
   /**
    * 分配用户权限的权限
    */
   String ASSIGN_ROLE = "AssignRole";
 
+
   /**
-   * namespace level permission
+   * namespace级别的权限
    */
-
   String MODIFY_NAMESPACE = "ModifyNamespace";
-
   String RELEASE_NAMESPACE = "ReleaseNamespace";
 
+  /**
+   * 查看配置的权限
+   */
+  String BROWSE_CONFIG = "BrowseConfig";
```

Comment:
I think it's better to name it as `VIEW_NAMESPACE` using the same format as `MODIFY_NAMESPACE` and `RELEASE_NAMESPACE `